cli/types/cypress-npm-api.d.ts

```diff
@@ -465,6 +466,31 @@ declare module 'cypress' {
     cypressVersion: string
     // TODO add resolved object to the configuration
     config: CypressConfiguration
+    /**
+     * If Cypress fails to run at all (for example, if there are no spec files to run),
+     * then it will set failures to 1 and will have actual error message in the
+     * property "message". Check this property before checking other properties.
+     *
+     * @type {number}
+     * @memberof CypressRunResult
```

Comment:
is this necessary? i don't get why this is needed if it's already in the `interface` declaration

Reply:
that's how JSDoc "document this" extension does it, maybe it is unnecessary, let me remove it